pages/manage/productions/create.vue

```diff
@@ -123,9 +131,9 @@
         return this.$i18n.locale;
       },
       suggestedEventIds() {
```

Comment:
```suggestion
      suggestedEventIdsWithoutProduction() {
```
Just a minor suggestion, to avoid confusion with `suggestedEventsWithProduction` (that one now seems a subset of `suggestedEventIds` to me at first glance, while it's actually two opposites)

Reply:
Remember to also change the usage(s)